src/workflows/workflow.py

```diff
@@ -531,3 +532,21 @@ def _validate(self) -> bool:
             InputRequiredEvent in produced_events
             or HumanResponseEvent in consumed_events
         )
+
+    def run_test(self) -> WorkflowTestRunner:
```

Comment:
I would prefer the Workflow api to not depend on the test runner. This wrapper is literally one line, I think having
```py
async with WorkflowTestRunner(workflow) as test_runner:
    ...
```
is ok?